src/middleware/validateFormMiddleware.ts

```diff
@@ -83,59 +83,20 @@ function getImpactValidation() {
 function getDateValidation() {
   return [
     body("day")
-      .optional({ checkFalsy: true })
       .isInt({ min: 1, max: 31 })
       .withMessage("Day is invalid")
       .escape(),
     body("month")
-      .optional({ checkFalsy: true })
       .isInt({ min: 1, max: 12 })
       .withMessage("Month is invalid")
       .escape(),
     body("year")
-      .optional({ checkFalsy: true })
       .isInt({
         min: new Date().getFullYear(),
         max: new Date().getFullYear() + 10,
       })
       .withMessage("Year is invalid")
       .escape(),
-    body(["day", "month", "year"]).custom((value, { req }) => {
-      const day = req.body.day;
-      const month = req.body.month;
-      const year = req.body.year;
-      const currentYear = new Date().getFullYear();
-
-      if (year && year < currentYear) {
-        throw new Error("Enter future date");
-      }
```

Comment:
This was a useful extra check, because the form now lets you input a date that was in the past...